DNN Platform/Library/Entities/Users/Users Online/AnonymousUserInfo.cs

```diff
@@ -29,7 +29,6 @@ namespace DotNetNuke.Entities.Users
     /// -----------------------------------------------------------------------------
     /// Project:    DotNetNuke
     /// Namespace:  DotNetNuke.Entities.Users
-    /// Class:      PurgeUsersOnline
     /// -----------------------------------------------------------------------------
     /// <summary>
     /// The AnonymousUserInfo class provides an Entity for an anonymous user
```

Comment:
This is in the `Users Online` directory, should it be marked as `Obsolete` as well?  It's only used from `DataProvider.UpdateUsersOnline`:
https://github.com/dnnsoftware/Dnn.Platform/blob/a1b217e038127f673fbfa82624e4d91f81993aab/DNN%20Platform/Library/Data/DataProvider.cs#L2498-L2519

Reply:
To add to this: this is in a folder called "Users Online" containing code that is just meant for this part. All classes should be marked obsolete IMHO. It can easily be implemented as a 3rd party sln external to the core. Since we're axing it, remove all related parts IMHO.

Reply:
I corrected the missing items